src/main/java/gov/nasa/jpl/mbee/mdk/options/MDKOptionsGroup.java

```diff
@@ -135,6 +139,29 @@ public void setUserScriptDirectory(String path) {
         addProperty(property, true);
     }
 
+    public List<String> getAuthenticationChain() {
```

Comment:
Can you help me understand the motivation for the chaining? Is there a scenario in which multiple authentication methods would be in use?

Reply:
Yes, we want to support multiple authentication methods in a configurable fashion